index.md

```diff
@@ -11,26 +11,67 @@ layout: default
 
 # Apache Beam (incubating)
 
-Apache Beam is an open source, unified model and set of language-specific SDKs for defining data processing workflows that may then be executed on top of a set of supported runners, currently including [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
+Apache Beam is an open source, unified programming model that you can use to define data processing workflows. You define a workflow, called a **pipeline**, by building a program with a Beam SDK. The pipeline is then executed by one of Beam's supported **distributed processing back-ends**, which include [Apache Flink](http://flink.apache.org), [Apache Spark](http://spark.apache.org), and [Google Cloud Dataflow](https://cloud.google.com/dataflow).
```

Comment:
The original wording said "workflow", which I assumed is because big data processing audiences may be more familiar with the term "workflow" while "pipeline" is more of a Beam term-of-art. I was thinking of the "define at first use" theory on communicating a new term.